css/style.css

```diff
@@ -60,7 +61,6 @@ form {
 #email {
   padding: 10px;
   margin-top: 20px;
-  border: none;
```

Comment:
- There is a duplicate of `border` in this property, it best practice not to duplicate values. There is a duplicate of `border` in this property, it best practice not to duplicate values.